Makefile.erb

```diff
@@ -0,0 +1,205 @@
+SHELL := /bin/bash
```

Comment:
I'm not keen on the Makefile being here when there is rake and it's only proxying to rake.